packages/core/lib/testing/testsource.js

```diff
@@ -24,9 +24,13 @@ TestSource.prototype.resolve = function(importPath, callback) {
 
       let abstractionFiles;
       try {
-        abstractionFiles = fse.readdirSync(
+        const buildDirFiles = (abstractionFiles = fse.readdirSync(
           self.config.contracts_build_directory
-        );
+        ));
+        // Filter out non-json files
+        abstractionFiles = buildDirFiles.filter(file => {
```

Comment:
Could/should this be `abstractionFiles = buildDirFiles.filter(file => file.match(/^.*.json$/);`? Its not particularly important, I just find it a little cleaner to do that if the only thing in the curly braces is the return statement.

Reply:
Sure, I just know that prettier is going to do something dumb-looking like
```
...buildDirFiles.filter(file =>
      file.match(....
```
heheh

Reply:
Haha, good point!

Reply:
I went ahead and changed it be a one-liner. Thanks @fainashalts!